content/installation/installing-manager.md

```diff
@@ -0,0 +1,104 @@
+
+{{% gsNote title="Administrator Password" %}}
+
+If you do not specify an administrator password, the installation process generates a random password and shows it as output when the installation is complete. We recommend that you specify an administrator password according to your security policy. 
+
+{{% /gsNote %}}
+
+You can install the [Cloudify CLI]({{< relref "installation/installing-cli.md" >}}) on a separate host to manage your Cloudify Manager remotely.
+
+## Installing Cloudify Manager
+
+_To install Cloudify Manager:_
+
+1. Go to the download page of the [Cloudify website](http://cloudify.co/download/).
+1. Copy the link address for the Cloudify Manager rpm file.
+1. From the CLI of your target host, run: ```sudo yum install \<rpm file link address>```
+   For example: ```sudo yum install http://cloudify-release-eu.s3.amazonaws.com/cloudify/4.3.0/release/cloudify-manager-install-4.3ga.x86_64.rpm```
+1. To change default configuration settings, edit the ```/etc/cloudify/config.yaml``` file.
+1. To install Cloudify Manager, run: ```cfy_manager install --private-ip <PRIVATE_IP> --public-ip <PUBLIC_IP>```
+
+  For more about the configure parameters, run: ```cfy_manager configure -h```
+
+## Other Installation Actions
+
+### Validating the Installation
+
+When the installation process is complete, you can use ```cfy status``` to make sure that all of the Cloudify services are running.
+Cloudify Premium customers can access the Cloudify Web Interface at: ```http://<manager_public_address>```
+
+An example output:
+{{< gsHighlight  sh >}}
+$ cfy status
+
+...
+
+Retrieving manager services status... [ip=127.0.0.1]
+
+Services:   
+   +--------------------------------+---------+
+   |            service             |  status |
+   +--------------------------------+---------+
+   | InfluxDB                       | running |
+   | Celery Management              | running |
+   | Logstash                       | running |
+   | RabbitMQ                       | running |
+   | AMQP InfluxDB                  | running |
+   | PostgreSQL                     | running |
+   | Manager Rest-Service           | running |
+   | Cloudify Stage                 | running |
+   | Webserver                      | running |
+   | Riemann                        | running |
+   | Webserver                      | running |
+   +--------------------------------+---------+
+
+   ...
+   {{< /gsHighlight >}}
+
+### Changing the Manager Parameters
+
+Before or after you install Cloudify Manager, you can change the parameters used in the installation. The configure command accepts the same CLI parameters as the install command, and it reads the config.yaml file for additional parameters.
```

Comment:
This is possibly only _after_ the initial install. You won't be able to run configure before install.

Reply:
Removed "Before or"